pkg/app/api/grpcapi/api.go

```diff
@@ -188,6 +190,59 @@ func (a *API) ListApplications(ctx context.Context, req *apiservice.ListApplicat
 			Value:    req.Disabled,
 		},
 	}
+
+	if req.EnvId != "" {
+		filters = append(filters, datastore.ListFilter{
+			Field:    "Id",
+			Operator: "==",
+			Value:    req.EnvId,
+		})
+	}
+	var envs []*model.Environment
```

Comment:
nit: This should be inside the `if` block.

Reply:
Actually, this declaration is not needed.

Reply:
ops, I forgot this 👍